File: 2727-number-of-senior-citizens/number-of-senior-citizens.cpp
class Solution {
public:
    int countSeniors(vector<string>& details) {
        int count = 0;
        for(auto i : details) {
            if(i[11] - '0' > 6) count++;
            else if (i[11] - '0' == 6 && i[12] - '0' > 0) count++;
        }
        return count;
    }
};

File: 2727-number-of-senior-citizens/number-of-senior-citizens.py
class Solution:
    def countSeniors(self, details: List[str]) -> int:
        count = 0
        for i in details:
            if int(i[11]) > 6:
                count += 1
            elif int(i[11]) == 6 and int(i[12]) > 0:
                count += 1
        return count